llvm/lib/Transforms/IPO/LowerTypeTests.cpp

```diff
@@ -95,6 +97,7 @@ STATISTIC(NumByteArraysCreated, "Number of byte arrays created");
 STATISTIC(NumTypeTestCallsLowered, "Number of type test calls lowered");
 STATISTIC(NumTypeIdDisjointSets, "Number of disjoint sets of type identifiers");
 
+namespace llvm {
```

Comment:
Why start the namespace llvm here? It doesn't need to go around these existing static cl::opt